### ISSUE TITLE
Possible to make it work with JavaScript?

### ISSUE THREAD

**Issue by Hrishikesh-K** [theNewDynamic/language-hugo-vscode#19]
Hello.

This extension works great for HTML files. Thank you for that. However, I was wondering if it's possible to be able to clear off those errors from my workspace when using Hugo snippets in JavaScript (via `<script>` tags or external JS files).

Currently, this is happening:

in HTML file:
![image](https://user-images.githubusercontent.com/11403551/100967385-17a34380-3555-11eb-897f-483db14fd09d.png)

in JS file:

![image](https://user-images.githubusercontent.com/11403551/100967452-3bff2000-3555-11eb-93f0-ebd958f7c005.png)

Because of this, I'm unable to use syntax and error highlighting of VSC as I don't realise if I have actually made a syntax error or if it's just because of Hugo's snippets. Thus, if you'd be able to add ability to highlight in these files, it'd be just amazing!

**Comment by larsendaniel** [comment]
Yes! I just came here to post the same issue and am happy to see someone else already reported it. It would be great if this could get fixed somehow.

**Comment by gonsen71** [comment]
Also CSS!!

**Comment by gonsen71** [comment]
Would be nice also to add it for CSS!

![Screenshot-2022-01-21T17:30:47](https://user-images.githubusercontent.com/85336518/150554660-16144b60-3da8-431c-9427-caac175dcc6c.png)

Also needs `resources.ExecuteAsTemplate` to process it.
